src/Resources/views/standard_layout.html.twig

```diff
@@ -173,15 +173,15 @@ file that was distributed with this source code.
                                                                         {% if menu.extra('safe_label', true) %}
                                                                             {{- label|raw -}}
                                                                         {% else %}
-                                                                            {{- label|truncate(100) -}}
+                                                                            {{- label|u.truncate(100) -}}
                                                                         {% endif %}
                                                                     </a>
                                                                 {% else %}
-                                                                    <span>{{ label|truncate(100) }}</span>
+                                                                    <span>{{ label|u.truncate(100) }}</span>
                                                                 {% endif %}
                                                             </li>
                                                         {% else %}
-                                                            <li class="active"><span>{{ label|truncate(100) }}</span></li>
+                                                            <li class="active"><span>{{ label|u.truncate(100) }}</span></li>
```

Comment:
Seems like there is other truncate to modify.
Cf: https://github.com/sonata-project/SonataAdminBundle/pull/6132/files

But you could wait for the PR to be merged.

BTW, is this change really CoreBundle related ?

Reply:
Truncate was move to SonataTwigBundle 0.1.1. It can be merge.

Reply:
Can we revert this change then ? 
There is already another PR for the migration truncate => u.truncate.
Let's avoid potential conflict

Reply:
I remove truncate commit.